dev-tools/audius-compose

```diff
@@ -161,7 +161,35 @@ def build(
 
 
 @cli.command()
-@click.option("-c", "--creator-node-replicas", default=4, type=int)
+@click.argument("service")
```

Comment:
i guess the only other potential issue i see here is if the underlying service.env file is changed any local changes might cause merge conflicts since it's tracked